File: 37-Java/src/Campo.java
public class Campo {
    protected int codice;
    protected String sport, nomeCampo;
    protected float larghezza, lunghezza, costoOrario;
    protected int prenotazioni;
    
    public Campo(int codice, String sport, String nomeCampo, float larghezza, float lunghezza, float costoOrario) {
        this.codice = codice;
        this.sport = sport;
        this.nomeCampo = nomeCampo;
        this.larghezza = larghezza;
        this.lunghezza = lunghezza;
        this.costoOrario = costoOrario;
        prenotazioni = 0;
    }

    public void addPrenotazione(){
        prenotazioni++;
    }

    public int getCodice() {
        return codice;
    }

    public float incasso(){
        return prenotazioni * costoOrario;
    }

    public String toString(){
        return sport +"\t"+ nomeCampo +"\t"+ codice +"\t"+ larghezza +"\t"+ lunghezza;
    }
}

File: 37-Python/Campo.py
class Campo:
    def __init__(self, codice, sport, nomeCampo, larghezza, lunghezza, costoOrario) -> None:
        self._codice = codice
        self._sport = sport
        self._nomeCampo = nomeCampo
        self._larghezza = larghezza
        self._lunghezza = lunghezza
        self._costoOrario = costoOrario
        self._prenotazioni = 0

    def addPrenotazioni(self):
        self._prenotazioni += 1
    
    def incasso(self):
        return self._prenotazioni * self._costoOrario

    def __str__(self) -> str:
        return self._sport +"\t"+ self._nomeCampo +"\t"+ str(self._codice) +"\t"+ str(self._larghezza) +"\t"+ str(self._lunghezza)